packages/node_modules/@webex/plugin-meetings/src/reconnection-manager/index.js

```diff
@@ -173,7 +186,7 @@ export default class ReconnectionManager {
    * @public
    * @memberof ReconnectionManager
    */
-  reconnect({networkChange = false, networkDisconnect = false}) {
+  reconnect({networkChange = false, networkDisconnect = false} = {networkChange: false, networkDisconnect: false}) {
```

Comment:
```suggestion
  reconnect({networkChange = false, networkDisconnect = false} = {}) {
```

Reply:
You don't need to define the object twice to get `reconnect()` to work.

Reply:
I was wondering about the same